extensions/sql-migration/src/sqlmigration.d.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface MigrationControllerProperties {
```

Comment:
If you aren't exposing this to other extensions I'd say that having a typings file like this is unnecessary.